article/helpers.py

```diff
@@ -56,3 +72,15 @@ def retrieve_articles(self):
         )
         response.raise_for_status()
         return [article['article_uuid'] for article in response.json()]
+
+    def article_read_count(self, category):
+        if category not in structure.CATEGORIES_LOOKUP:
+            raise Exception('Incorrect category')
+
+        read_articles = frozenset(self.retrieve_articles())
+        articles_in_category = frozenset(
+            [article.uuid for article in
+             structure.CATEGORIES_LOOKUP[category].articles]
+        )
+        read_articles_in_category = read_articles & articles_in_category
```

Comment:
`read_articles_in_category = read_articles.intersection(articles_in_category)` would be much more explicit, I'd even consider adding a comment that `read_articles_in_category is a new set with elements common to read_articles and articles_in_category`

Reply:
This time I strongly don't agree. '&' is the intersection operator in Python, how `intersection()` would be more explicit? Then we should use `__add__` instead of `+`?